src/script/assets/AssetURLCache.ts

```diff
@@ -31,11 +29,8 @@ export const setAssetUrl = (identifier: string, url: string) => {
     return isExistingUrl;
   }
 
-  const outdatedUrl = cache.set(identifier, url);
-
-  if (outdatedUrl != null) {
-    window.URL.revokeObjectURL(outdatedUrl);
-  }
+  cache.set(identifier, url);
+  window.URL.revokeObjectURL(url);
```

Comment:
That sounds wrong. Revoking the URL after setting it, that seems weird 🙃

Reply:
Isn't it what we've been doing before also? 🤔